Makefile

```diff
@@ -293,6 +293,17 @@ test-db-migration: start-docker ## Gets diff of upgrade vs new instance schemas.
 	./scripts/mysql-migration-test.sh
 	./scripts/psql-migration-test.sh
 
+gomodtidy:
+	@cp go.mod go.mod.orig
+	@cp go.sum go.sum.orig
+	$(GO) mod tidy
+	@if [ "$$(diff go.mod go.mod.orig)" != "" -o "$$(diff go.sum go.sum.orig)" != "" ]; then \
+		echo "go.mod/go.sum was modified. \ndiff- $$(diff go.mod go.mod.orig) \n$$(diff go.sum go.sum.orig) \nRun \"go mod tidy\"."; \
```

Comment:
Apparently `\n` doesn't work here:
```bash
go.mod/go.sum was modified. \ndiff- 44c44
< 	github.com/jonboulle/clockwork v0.1.0
---
> 	github.com/jonboulle/clockwork v0.1.0 // indirect
69d68
< 	github.com/prometheus/client_model v0.2.0 \n5,6d4
< contrib.go.opencensus.io/exporter/ocagent v0.4.9/go.mod h1:ueLzZcP7LPhPulEBukGn4aLh7Mx9YJwpVJ9nL2FYltw=
< git.apache.org/thrift.git v0.0.0-20180902110319-2566ecd5d999/go.mod h1:fPE2ZNJGynbRyZ4dJvy6G277gSllfV2HJqblrnkyeyg= \nRun "go mod tidy".
```

Reply:
Strange, this is what I see:
```
go mod tidy
warning: ignoring symlink /home/agniva/mattermost/mattermost-server/client
warning: ignoring symlink /home/agniva/mattermost/mattermost-server/enterprise
go.mod/go.sum was modified. 
diff- 44c44
< 	github.com/jonboulle/clockwork v0.1.0
---
> 	github.com/jonboulle/clockwork v0.1.0 // indirect
69d68
< 	github.com/prometheus/client_model v0.2.0 
 
Run "go mod tidy".
make: *** [Makefile:300: gomodtidy] Error 1
```

Reply:
What shell are you running? I'm using `bash`

Reply:
What shell are you running? I'm using `bash`

Reply:
I'm also running bash.